services/web/server/src/simcore_service_webserver/statics_settings.py

```diff
@@ -18,6 +18,27 @@
 assert FRONTEND_APP_DEFAULT in FRONTEND_APPS_AVAILABLE
 
 
+class OsparcService(BaseModel):
```

Comment:
```suggestion
class OsparcDependency(BaseModel):
```
My bad here, could you kindly rename it?